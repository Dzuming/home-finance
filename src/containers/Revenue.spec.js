import React from 'react';
import { create } from 'react-test-renderer';
import Reveue from './Revenue';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import thunk from 'redux-thunk';

const middleware = [thunk];
const mockStore = configureMockStore(middleware);
const store = mockStore({
  revenue: 55,
  user: [],
  financeFlow: { selectedDate: '2018-03' },
  login: { isAuthenticated: true },
});
const component = create(
  <Provider store={store}>
    <Reveue />
  </Provider>,
);

describe('<Reveue />', () => {
  it('should match snapshot', () => {
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
