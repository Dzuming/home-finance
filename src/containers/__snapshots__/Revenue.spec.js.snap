// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Reveue /> should match snapshot 1`] = `
<div>
  55
   z≈Ç
</div>
`;
